public/html/projects/edit.pug

```diff
@@ -7,8 +7,12 @@ form.form-horizontal
 			input.form-control(type="text" ng-model="projectName" placeholder="Project Name")
 
 	.form-group
+		label.control-label.col-sm-4 Allow alerts for this project
+		.col-sm-1: input.form-control(type="checkbox" title="Send email alerts about failed tasks" ng-model="alert")
```

Comment:
See the docs for checkboxes in bootstrap: [`.checkbox` class](https://getbootstrap.com/css/#forms-example)